edxval/api.py

```diff
@@ -104,6 +105,35 @@ def create_video(video_data):
         raise ValCannotCreateError(serializer.errors)
 
 
+def update_video(video_data):
+    """
+    Called on to update Video objects in the database
+
+    update_video is used to update Video objects.
```

Comment:
Could be clearer in the doc. 

 `update_video is used to update Video objects by the given edx_video_id.`

I think we may want to add some Raise docs depending on the comments below.